pkg/dyninst/testdata/decoded/sample/testMapIntToInt.json

```diff
@@ -0,0 +1,42 @@
+[
+  {
+    "service": "sample",
+    "ddsource": "dd_debugger",
+    "logger": {
+      "name": "",
+      "method": "main.testMapIntToInt",
+      "version": 0,
+      "thread_id": 0,
+      "thread_name": ""
+    },
+    "debugger": {
+      "snapshot": {
+        "id": "[id]",
+        "timestamp": "[ts]",
+        "language": "go",
+        "stack": "[stack-unredact-me]",
+        "probe": {
+          "id": "testMapIntToInt",
+          "location": {
+            "method": "main.testMapIntToInt"
+          }
+        },
+        "captures": {
+          "entry": {
+            "arguments": {
+              "m": {
+                "type": "map[int]int",
+                "size": 10,
+                "entries": [
+                  "notCapturedReason",
+                  "pruned"
+                ]
+              }
```

Comment:
what's up with this one?

Reply:
It appear the group data isn't present in the event's data items. Investigating.